exectors/nodejs/numberformat.js

```diff
@@ -0,0 +1,76 @@
+// Tests NumberFormat and DecimalFormat classes
+
+  let patternsToOptions = new Map();
+  patternsToOptions.set("0.0", {minimumFractionDigits: 1});
+  patternsToOptions.set("00", {minimumSignificantDigits: 1});
+  patternsToOptions.set("0.0", {minimumFractionDigits: 1});
```

Comment:
Duplicate of line 4. Either way, the mapping takes place in the data generator and this map can be deleted altogether.

Reply:
Removed the duplicate line. Yes, I'll remove this function when I get the options working.